src/backend/binaryen/wasm_expr_gen.ts

```diff
@@ -1083,15 +1080,6 @@ export class WASMExpressionGen {
                         );
                     }
                 }
-            } else {
```

Comment:
Is this code useless?

Reply:
Yeah, I think that we used to add the function `getInfcMember` here as a workaround, based on our test samples result, `interfaceObj.xx` will never become a `OffsetCall`.